### ISSUE TITLE
Music changes automatically in Continuous mode

### ISSUE THREAD

**Issue by AnonymouV** [djyt/cannonball#153]
Hello everyone, 

Not sure if I'll get an answer but something has been bothering me since I've updated Cannonball to the last version.

I mostly play the Continuous mode and everytime I reach the Desert stage (stage 6 in Continuous mode), the current custom song stops playing and the original Magical Sound Shower starts. If I stop my Testarossa before the Desert Stage Checkpoint, the custom song keeps playing, but as soon as I pass the checkpoint, it's over...

Anyone else having the issue?  If so, found a fix?
Is there any way to get rid of the inbuilt songs?

Thank you guys and sorry for my bad English. Not really my native language, as you can see.

Edit: Finally rolled back to a previous version where I could replace the inbuilt soundtrack. Works like a charm.
Hopefully it will be fixed in the future! In the meantime, thank you very much @djyt for the incredible work!
I've been enjoying Cannonball since 2016 and I'm planning on speedrunning it now. Out Run holds a special place in my heart and being able to play it like that is absolutely perfect.

Wishing you the best mate!

**Comment by JoeSkee** [comment]
Yes, I am experiencing the same issue when in continuous mode. It changes the custom song I am playing to the built in songs. It is annoying and hopefully an option can be added to toggle that on/off. Otherwise this game is great!

**Comment by GallopRacer** [comment]
>rolled back to a previous version where I could replace the inbuilt soundtrack

0.32 ftw

however, the time must be set to OFF, because in this version when continuous mode is used checkpoints do not refill time
